internal/service/boot.go

```diff
@@ -20,6 +21,15 @@ import (
 	"github.com/elastic/elastic-package/internal/testrunner/runners/system"
 )
 
+const (
+	testRunMaxID = 99999
+	testRunMinID = 10000
+)
+
+func createTestRunID() string {
+	return fmt.Sprintf("%d", rand.Intn(testRunMaxID-testRunMinID)+testRunMinID)
+}
+
```

Comment:
Copied from testrunner/system. 

WDYT about dding this to `internal/common/helpers.go` ?

Reply:
Yes, if used in multiple places we could move it to some common package.

Reply:
Ok, I'll move it to that location then.